### ISSUE TITLE
implicit cast spamming logs when sleeping

### ISSUE THREAD

**Issue by ainesophaur** [wire-elements/wire-spy#26]
My console logs started to get spammed with the following message:

> Implicit conversion from float 0.25 to int loses precision in vendor/wire-elements/wire-spy/src/SupportHotReloading.php on line 112

The code causing the issue is 

```php
sleep(.25);
```

I fixed it locally by changing the sleep function to `usleep`

```php
usleep(25000);
```

**Comment by Cegem-360** [comment]
Yes, this issue is solved by me and Merged into code, but i miss calculate the value instead of 250 => 250000 = 0.25 second like sleep(0.25)

**Comment by ainesophaur** [comment]
I checked the update and I see you're doing 

`usleep(250)` which will now run 4000 times a second (0.0025 seconds).. just making sure that was the intention as that seems a bit high and could probably be increased to have fewer cycles.

**Comment by PhiloNL** [comment]
Added `usleep(250000);` in `v0.0.10`
